locksmith/src/utils/lockMigrate.ts

```diff
@@ -0,0 +1,260 @@
+  const keyManagersChanges = keyEvents.filter(
+    ({ event }: any) => event === 'KeyManagerChanged'
+  )
+  migrateLogEvent.emit('migrateLock', {
+    recordId,
+    msg: `CLONE LOCK > ${transfers.length} keys transferred, ${keyManagersChanges.length} key managers changed`,
+  })
+
+  migrateLogEvent.emit('migrateLock', {
+    recordId,
+    msg: 'CLONE LOCK > fetching managers...',
+  })
+
+  // fetch managers from graph
+  if (subgraphURI) {
+    let managers
+    try {
+      managers = await listManagers({
+        lockAddress: newLockAddress,
+        subgraphURI,
+      })
+    } catch (error: any) {
+      migrateLogEvent.emit('migrateLock', {
+        recordId,
+        msg: error.message,
+      })
+      managers = []
+    }
+    if (managers.length) {
+      migrateLogEvent.emit('migrateLock', {
+        recordId,
+        msg: `LOCK > managers for the lock '${await newLock.name()}':`,
+      })
+      managers.forEach((account: string, i: number) => {
+        migrateLogEvent.emit('migrateLock', {
+          recordId,
+          msg: `[${i}]: ${account}`,
+        })
+      })
+
+      const txs = await Promise.all(
+        managers.map((manager: string) => newLock.addLockManager(manager))
+      )
+      const waits = await Promise.all(txs.map((tx: any) => tx.wait()))
+      waits.forEach(({ events }) => {
+        const evt = events.find((evt: any) => evt.event === 'LockManagerAdded')
+        migrateLogEvent.emit('migrateLock', {
+          recordId,
+          msg: `LOCK CLONE > ${evt.args.account} added as lock manager.`,
+        })
+      })
+    }
+  } else {
+    migrateLogEvent.emit('migrateLock', {
+      recordId,
+      msg: 'Missing SubgraphURI. Can not fetch from The Graph on this network, sorry.',
+    })
+  }
+
+  // update the beneficiary
+  const txBenef = await newLock.updateBeneficiary(beneficiary)
+  await txBenef.wait()
+  migrateLogEvent.emit('migrateLock', {
+    recordId,
+    msg: `LOCK CLONE > ${beneficiary} set as beneficiary.`,
+  })
+
+  // update metadata
+  if (symbol != 'UDT') {
+    const txSymbol = await newLock.updateLockSymbol(symbol)
+    await txSymbol.wait()
+    migrateLogEvent.emit('migrateLock', {
+      recordId,
+      msg: `LOCK CLONE > Symbol updated to '${symbol}'.`,
+    })
+  }
+
+  // tokenURI
+  try {
+    // check if is set to default or not
+    const tokenURI = new URL(tokenURISample)
+    if (!tokenURI.host.includes('unlock-protocol.com')) {
+      const totalSupply = keyOwners.length
+      const baseTokenURI = tokenURISample.slice(0, -`${totalSupply}`.length)
+      await newLock.setBaseTokenURI(baseTokenURI)
+      migrateLogEvent.emit('migrateLock', {
+        recordId,
+        msg: `LOCK CLONE > baseTokenURI updated to '${baseTokenURI}'.`,
+      })
+    }
+  } catch (error) {
+    migrateLogEvent.emit('migrateLock', {
+      recordId,
+      msg: `LOCK CLONE > baseTokenURI not set, using default. (ex. '${tokenURISample}').`,
+    })
+  }
+
+  if (symbol != 'UDT') {
```

Comment:
```suggestion
  if (symbol != 'KEY') {
```

UDT is the symbole of the Governance token. The symbol for the NFT is`KEY` :)